tests/unit/serializer-test-helpers.js

```diff
@@ -0,0 +1,84 @@
+
+const appRoot = require('app-root-path');
+const { expect } = require('chai');
+const _ = require('lodash');
+
+const { openapi } = appRoot.require('utils/load-openapi');
+const { fakeId, fakeBaseUrl } = appRoot.require('tests/unit/test-data');
+
+const resourceSubsetSchema = (resourceType, resourceAttributes) => {
+  const fakeUrl = `${fakeBaseUrl}/${resourceType}s/fakeId`;
+  const schema = {
+    links: {
+      self: fakeUrl,
+    },
+    data: {
+      id: fakeId,
+      type: resourceType,
+      links: { self: fakeUrl },
+    },
+  };
+  if (resourceAttributes) {
+    schema.data.attributes = resourceAttributes;
+  }
+  return schema;
+};
+
+/**
+ * @summary Helper function for lite-testing single resource
+ * @function
+ * @param {object} serializedResource serialized resource
+ * @param {string} resourceType resource type
+ * @param {object} nestedProps object containing properties nested under data.attributes
+ */
+const testSingleResource = (serializedResource, resourceType, nestedProps) => {
+  expect(serializedResource).to.containSubset(resourceSubsetSchema(resourceType, nestedProps));
+
+  if (nestedProps) {
+    _.forEach(Object.keys(nestedProps), (prop) => {
+      expect(serializedResource).to.have.nested.property(`data.attributes.${prop}`);
+    });
+  }
+};
+
+/**
+ * @summary Helper function for lite-testing multiple resources
+ * @function
+ * @param {Object} serializedResources serialized resources
+ * @returns {Object} data object from serialized resources for further use
```

Comment:
Could you rename `Object` to `object`? This will be our convention soon based on [this](https://github.com/gajus/eslint-plugin-jsdoc#why-not-capital-case-everything). Could you also make this change on the rest of your functions?